decisions/size.go

```diff
@@ -0,0 +1,10 @@
+package decisions
+
+// Size returns the total size of the tree, including this node.
+func (n *Node) Size() int {
+	if n.IsAction() {
+		return 1
+	}
+
+	return 1 + n.YesNode.Size() + n.NoNode.Size()
+}
```

Comment:
I'll probably just put this in utils.go unless there's a lot of other size-specific stuff we need to do later

Reply:
Yeah up to you. I like to keep files small in Go since the test files correspond to the code files and there's not a lot of good ways to organize the tests.

Reply:
Ah, gotcha, okay